### DISCUSSION TITLE
Is there possible change the mouse pointer(cursor) when mouse moving into wezterm?

### DISCUSSION THREAD

**Discussion by zw963** [wezterm/wezterm#5274] [Q&A]
### What Operating System(s) are you running on?

Linux Wayland

### Which Wayland compositor or X11 Window manager(s) are you using?

mutter

### WezTerm version

20240203-110809-5046fc22

### Ask your question!

Let me upload a screenshot for describe my issue.

![image](https://github.com/wez/wezterm/assets/549126/b83acbba-9b8a-489e-88aa-f3f777aeed13)

as you can see, i update to use a new theme recent days.
But, i almost can't found where is my mouse point because it too hard to find it.

is there exists config for style this point? e.g. more bigger, or other shape?

Thanks

**Comment by zw963** [comment]
Anyone can help? Thanks, this issue annoy me a lot for a long time

**Comment by chtenb** [comment]
Isn't that something you should configure in your OS? Or do you want to have wezterm use a different mouse pointer than your other applications?

**Comment by wez** [comment]
The cursor is provided by your OS.  On X11 and Wayland systems, you can influence this via some environment variables, such as `XCURSOR_THEME`, `XCURSOR_PATH` and `XCURSOR_SIZE`. See https://wezfurlong.org/wezterm/faq.html#i-use-x11-or-wayland-and-my-mouse-cursor-theme-doesnt-seem-to-work

**Reply by zw963** [reply]
> Isn't that something you should configure in your OS? Or do you want to have wezterm use a different mouse pointer than your other applications?

Thanks for help, yes, i can change my mouse point in GNOME + Wayland use `Appearance -> Cursor` settings (please check following screenshot)

![image](https://github.com/user-attachments/assets/1ad6855d-61ff-4208-80bc-36a73b2a1cd3)

e.g. when i change to Oxygen_Blue, the cursor color change to blue, it works!

But, the issue is, after the mouse pointer moving into wezterm, the color always rollback into the `I`  shape and default color (as the screnshot in my question post),  It often spend several seconds to found where is the mouse pointer is.

>  such as XCURSOR_THEME, XCURSOR_PATH and XCURSOR_SIZE. See https://wezfurlong.org/wezterm/faq.html#i-use-x11-or-wayland-and-my-mouse-cursor-theme-doesnt-seem-to-work

Hi, @wez , i checked the above doc, and try run wezterm use `XCURSOR_THEME=Oxygen_Blue wezterm` OR add config like:

```lua
config.set_environment_variables = {
  SHELL="/usr/bin/bash",
  XCURSOR_THEME="Oxygen_Blue", 
  LS_COLORS='',
}
```

Both of them not work, when mouse pointer moving into wezterm window, keeping the `I` and hard to find.

Thanks

**Reply by chtenb** [reply]
Is the issue specific to wezterm, or does this always happen when hovering over text? E.g. like in text editors

**Reply by zw963** [reply]
> Is the issue specific to wezterm, or does this always happen when hovering over text? E.g. like in text editors

I just found, it's seem like this always happen if mouse pointer hovering over text, except, the `I` is look very difficult when current window is wezterm (fullscreen), probably because my theme ?

Could you help on how to solve this?

Thanks

**Reply by chtenb** [reply]
I can't, because I'm not on your system. I would ask around on a forum that is specifically about the OS you're using, if you can't figure it out yourself
